src/System.Net.Security/tests/FunctionalTests/unix/project.json

```diff
@@ -29,7 +29,8 @@
     "Microsoft.DotNet.BuildTools.TestSuite": "1.0.1-prerelease-01001-04"
   },
   "frameworks": {
-    "netstandard1.7": {}
+    "netstandard1.7": {},
+    "netcoreapp1.1": {}
```

Comment:
Thanks @weshaggard. Please explain to avoid this in the future.
Why is my local and the CI build not catching this?

/cc: @karelz @ericstj